components/cms-services/deploy/asyncapi/Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM alpine:3.10
+FROM gcr.io/distroless/static:latest
```

Comment:
`:latest`?

Reply:
Yes, there are no semver tags for this image :(. We are also using `latest` in controllers ;)